### PR TITLE
Validate subscription activation key

### PR DESCRIPTION
**Pull request by jkozol**
If register now is selected, an activation key is required. If no activation key is entered the Next and Create buttons are disabled.
Also, if the activation key is empty an error message will appear to prompt the user to enter an activation key.

The subscription errors variable is removed in favor of declaring the error message directly in the registration step component. The validateSubscription function is removed in favor of validating the activation key field on change.

This fixes #114 and is dependent on #112 

invalidActivationKey

### CODE REVIEW

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ WizardStepReview.propTypes = {
     subscription: PropTypes.object,
     subscribeNow: PropTypes.bool,
     uploadAWSErrors: PropTypes.object,
-    subscriptionErrors: PropTypes.object,
+    isSubscriptionValid: PropTypes.object,
```
Comment (Gundersanne) [inline src/PresentationalComponents/CreateImageWizard/WizardStepReview.js:133]: Hm should this be `isValidSubscription`?

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ const WizardStepReview = (props) => {
     return (
         <>
             { (Object.keys(props.uploadAWSErrors).length > 0 ||
-               Object.keys(props.subscriptionErrors).length > 0) &&
+               !props.isSubscriptionValid) &&
```
Comment (Gundersanne) [inline src/PresentationalComponents/CreateImageWizard/WizardStepReview.js:99]: The property passed from the wizard is `isValidSubscription` no?

**Review by Gundersanne — COMMENTED** [review]
Sorry for the late review, have a q.

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.js (hunk 1 above)
Comment (jkozol) [reply src/PresentationalComponents/CreateImageWizard/WizardStepReview.js]: You are totally correct. Thanks for catching this. Fixed now.

File: src/PresentationalComponents/CreateImageWizard/WizardStepReview.js (hunk 2 above)
Comment (jkozol) [reply src/PresentationalComponents/CreateImageWizard/WizardStepReview.js]: Fixed.

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**